### PR TITLE
Debug

### PR DESCRIPTION
**Pull request by flov3rh**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,10 +2,85 @@
...
+    </main>
+   
+    <section class="promo">
+      <div class="flex_container">
+        <figure>
+          <a href=""><img src="assets\img\promo_iphone5s.jpg" alt=""/></a>
+        </figure>
+
+        <figure>
+          <a href=""><img src="assets\img\promo_iphone5c.jpg" alt=""/></a>
```
Comment (Ispirett) [inline index.html]: For this section 
- [x] Each link is an <a> HTML tag, with a <div> tag inside it.
- [x] The` <div>` tag  inside the `<a> `tag should use an image as background 

- [x] The text on the original page is actually an image

File: index.html [hunk 2]
```diff
@@ -2,10 +2,85 @@
...
+          Shop the <span><a href="" class="foot_link">Apple Online Store</a></span> (1-800-MY-APPLE), visit an <span><a href="" class="foot_link">Apple Retail Store</a></span>, or find a <span><a href="" class="foot_link">reseller</a></span>.
+        </p>
+
+        <p>
+          Copyright &#169; 2014 Apple Inc. All rights reserved. <span><a id="terms" href="" class="foot_link">Terms of Use</span></a><span><a href="" class="foot_link">Privacy Policy</a></span>
+        </p>
+      </div>
+
+      <ul class="flex_container">
+        <li><a href="" class="foot_link">Site Map<a/></li>
```
Comment (Ispirett) [inline index.html:67]: For the links :
inside the`<ul>`
use *display flex* for positioning links.

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey, @flov3rh  great work with this project you are doing a fantastic job :1st_place_medal: . There are just a few changes that Mircoverse requires before this project can be completed.

- [x] **Optional** 
The main section under the nav should be images instead of text.

File: index.html (hunk 2 above)
Comment (flov3rh) [reply index.html]: I don't understand this suggestion right here

**Comment by mosaaleb** [comment]
Reviewing

File: index.html [hunk 3]
```diff
@@ -2,10 +2,81 @@
...
+              <li><input class="gradient" type="search" name="search"/></li>
+            </ul>
+          </nav>
+        </div>
+
+        <div class="content flex_container">
+           <a href="">
+            <figure class="content_header">
+              <img src="assets/img/your_verse_title_small.png"/>
+            </figure>
```
Comment (mosaaleb) [inline index.html]: `content flex_container` part:
- [x] There should be three links
- [x] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
- [x] The `<div>` tag  inside the `<a>` tag should use an image as background -- [when to use background image vs image tag](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)

File: index.html [hunk 4]
```diff
@@ -2,10 +2,81 @@
...
+           </a>
+        </div>
+      </div>
+    </main>
+   
+    <section class="promo">
+      <div class="flex_container">
+          <a href=""><div class="picture_1 footer_img"></div></a>
+          <a href=""><div class="picture_2 footer_img"></div></a>
+          <a href=""><div class="picture_3 footer_img"></div></a>
```
Comment (mosaaleb) [inline index.html]: The same concept of img tag vs background-image also applies here:
- [x]  Promo links section should be made of four `<a>` tags with `<div>` tags inside it
- [x] The `<div>`s inside the `<a>` tags should use an image as a background

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @flov3rh 
#### Amazing work :clap:. Most of requirements are done perfectly. The UI looks very clean. Yet there is minor requirement related the use of `html` `` tag vs the `css` `background-image` property. I have put a link to understand the difference and when to use either of them in the comments below.

Please fix these minor issue and submit a new code review

## Happy Coding :computer:

**Comment by mosaaleb** [comment]
reviewing

**Review by mosaaleb — APPROVED** [review]
### LGTM :+1:

Congratulations @flov3rh .You can merge now to master.

Good luck with the next projects.

**Pull request merged**